### ISSUE TITLE
Pasting text with newlines adds additional newlines

### ISSUE THREAD

**Issue by EbonJaeger** [contour-terminal/contour#753]
### Contour Terminal version

0.3.2 - Commit: 0e6d75a2042437084c9f9880a5c8b5661a02da07

### Installer source

Github: source code cloned

### Operating System

Linux (Solus 4.3)

### Architecture

x86-64

### Other Software

nano 6.3
GEdit 42.1

### Steps to reproduce

1. Write some text in gedit containing newlines
2. Select all text and copy
3. Paste text in nano

### Expected Behavior

No extra newlines should be added.

### Actual Behavior

It looks like two newlines are added for each single newline in the pasted text.

### Additional notes

A picture is worth a thousand words, so hopefully this clearly demonstrates what I'm talking about :smiley: 
![image](https://user-images.githubusercontent.com/5157277/175823786-c2686452-48a7-4569-9f30-458e95d06ba1.png)

**Comment by Utkarsh-khambra** [comment]
Thanks for the bug report. I tested this, and can reproduce it with gedit and contour. Also I found that alacritty and kitty also behave the same as contour, while gnome-terminal and xterm do not add newlines.Maybe either gedit on copy adds more characters than just a \n or contour is interpreting the clipboard data incorrectly.

One more thing `wl-paste` seems to paste lines correctly in contour.
